_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -0,0 +1,241 @@
+  every hop along a payments path secures its conditional payment with
+  the same hash digest.  This means that a user who controls two nodes
+  along the same path knows that any hops between those nodes were not
+  the ultimate spender or receiver of the payment.  This not only
+  reduces the amount of privacy provided by LN's onion routing but it
+  also allows a malicious user to steal the routing fees paid to the
+  in-between hops (this is known as the [wormhole attack][]).  For
+  example, in the following route, Mallory can steal Bob and Carol's
+  routing fees as well as conclude that neither of them is the spender
+  or receiver of the ultimate payment.
+
+      Alice → Mallory → Bob → Carol → Mallory' → Dan
+
+    PTLCs make it possible for each hop to use a different identifier
+    for the payment by using adaptor signatures (which represent
+    *points* on an elliptic curve) rather than hashes.  Adaptor
+    signatures were originally described for use with the [schnorr
+    signature scheme][topic schnorr signatures].  It's known to be
+    possible to use them with Bitcoin's current ECDSA signature scheme
+    (see [Newsletter #16][news16 2pecdsa scriptless]) but the process
+    relies on two-party ECDSA signing (2pECDSA) which is complex and
+    requires security assumptions beyond those normally required for
+    Bitcoin-style ECDSA signatures.  However, more recently, Lloyd
+    Fournier published a [paper][fournier otves] describing how to
+    securely use adaptor signatures using just regular 2-of-2 Bitcoin
+    multisig (e.g.  `OP_CHECKMULTISIG`) and simple discrete log
+    equivalence (DLEQ); this was summarized in a [post][uSEkaCIO email]
+    to the Lightning-Dev mailing list last November.
+
+    Last week during the [Lightning HackSprint][], several developers
+    [worked][ptlc challenge] on implementing these 2-of-2 multisig
+    adaptor signatures.  The result was rough code for it being written
+    for the C-language [libsecp256k1][jonasnick otves] and Scala [bitcoin-s][nkohen otves] libraries.
+    This is currently nowhere near ready for production, but it can help
+    developers begin experimenting with the use of adaptor signatures on
+    mainnet, both for PTLCs in LN and for use in other trustless
+    contract protocols.
+
+## Bitcoin Core PR Review Club
+
+_In this section, we summarize a recent Bitcoin Core PR Review Club meeting,
+highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting._
+
+[Retry notfounds with more urgency][review club 18238] is a PR
+([#18238][Bitcoin Core #18238]) by Anthony Towns that would change peer-to-peer
+behavior so that when nodes receive a `notfound` in response to a request for a
+transaction, they would skip the current time-out period and instead try
+to obtain the transaction as fast as possible from another peer.
+
+Discussion began with fundamental reasons for the PR:
```

Comment:
Should we mention the host? e.g. "Discussion was led by [amiti](github url) and began with..."

Reply:
http://bitcoinops.org/en/newsletters/2020/03/11/#bitcoin-core-pr-review-club didn't mention the host (which looks like it was also hosted by Uttarwar).  I have no preference either way.